modules/web/src/app/node-data/component.ts

```diff
@@ -691,7 +721,14 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
     };
   }
 
-  private _isClusterAutoscalingAppExist(apps: ApplicationDefinition[]): boolean {
-    return !!apps.find(app => app.name === CLUSTER_AUTOSCALING_APP_DEF_NAME);
+  private _filterOutAutoscalerApp(updateApplication: boolean, app: Application): void {
+    this._applicationService.applications = this._applicationService.applications.filter(
+      app => app.spec.applicationRef.name !== CLUSTER_AUTOSCALING_APP_DEF_NAME
+    );
+
+    if (updateApplication) {
```

Comment:
This should not be part of this method since this method's purpose is to filter out the autoscaler app, not add it back.